x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/documents.test.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { setMockValues } from '../../../__mocks__/kea.mock';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { EuiPageHeader, EuiCallOut } from '@elastic/eui';
+
+import { DocumentCreationButton } from './document_creation_button';
+import { SearchExperience } from './search_experience';
+import { Documents } from '.';
+
+describe('Documents', () => {
+  const values = {
+    isMetaEngine: false,
+    myRole: { canManageEngineDocuments: true },
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    setMockValues(values);
+  });
+
+  it('renders', () => {
+    const wrapper = shallow(<Documents engineBreadcrumb={['test']} />);
+    expect(wrapper.find(EuiPageHeader).length).toEqual(1);
+  });
+
+  it('renders a DocumentCreationButton if the user can manage engine documents', () => {
+    setMockValues({
+      ...values,
+      myRole: { canManageEngineDocuments: true },
+    });
+
+    const wrapper = shallow(<Documents engineBreadcrumb={['test']} />);
+    expect(wrapper.find(DocumentCreationButton).length).toEqual(1);
+  });
+
+  it('renders a SearchExperience', () => {
+    const wrapper = shallow(<Documents engineBreadcrumb={['test']} />);
+    expect(wrapper.find(SearchExperience).length).toEqual(1);
```

Comment:
I think you'd be fine moving this (`expect(wrapper.find(SearchExperience)`) to the main `it renders` test

Reply:
f9e01c33880a3fc512baca354dfca1d19a06e7e7